src/Analyzers/Microsoft.AspNetCore.Analyzer.Testing/src/Microsoft.AspNetCore.Analyzer.Testing.csproj

```diff
@@ -2,7 +2,7 @@
 
   <PropertyGroup>
     <Description>Helpers for writing tests for Roslyn analyzers.</Description>
-    <TargetFramework>netstandard2.0</TargetFramework>
+    <TargetFrameworks>$(DefaultNetFxTargetFramework);netstandard2.0</TargetFrameworks>
```

Comment:
Same thing. This is a test only project, so unless something requires it, I wouldn't cross-compile it.